habitat/sims/rearrangement/actions.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import List
+
+import attr
+import magnum as mn
+
+import habitat_sim
+from habitat.core.registry import registry
+from habitat.sims.habitat_simulator.actions import (
+    HabitatSimActions,
+    HabitatSimV1ActionSpaceConfiguration,
+)
+from habitat_sim.agent.controls.controls import ActuationSpec
+
+
+def raycast(sim, sensor_name, crosshair_pos=[128, 128], max_distance=2.0):
```

Comment:
This utility is great, so great that maybe it should live in a higher level? Also it should be under sims/habitat_sim/ since it relies heavily on habitat_sim specific APIs, right?